### DISCUSSION TITLE
Orientation.

### DISCUSSION THREAD

**Discussion by kizzle4shaw** [PMEAL/porespy#819] [Q&A]
I used the snow2 to extract the estaillaides pore network. the image is 1000*3. After extraction the network has been viewed in paraview and its a little odd that the x axis seems smaller than z while the cube is of similar dimensional magnitude. Is this normal?

![image](https://user-images.githubusercontent.com/32852010/229573866-46e14192-539f-486b-b7a7-b6f7b682e675.png)

**Comment by jgostick** [comment]
That depends on your image.  Are the pores anisotropic?  Or perhaps voxel size is not equal in all directions.
